sshtun.go

```diff
@@ -203,11 +203,18 @@ func (tun *SSHTun) Start() error {
 		if err != nil {
 			return tun.errNotStarted(fmt.Errorf("Error reading SSH key file %s: %s", tun.keyFile, err.Error()))
 		}
-		key, err := ssh.ParsePrivateKey(buf)
+		var key ssh.Signer
+		if tun.password == "" {
+			key, err = ssh.ParsePrivateKey(buf)
+		} else {
+			key, err = ssh.ParsePrivateKeyWithPassphrase(buf, []byte(tun.password))
```

Comment:
An empty passphrase will still return an unencrypted key, so this condition may not be necessary.